src/errors.rs

```diff
@@ -141,6 +141,11 @@ pub enum RustupError {
         target: TargetTriple,
         suggestion: Option<String>,
     },
+    #[error(
+        "unable to correctly link proxies. this might be a bug in rustup, please open a new issue here:\n\
```

Comment:
The usual trick for style consistency here is that you don't write dots. Probably:

```rs
"unable to correctly link proxies\nnote: this might be a bug in rustup, please file an issue:\nnote: https://github.com/rust-lang/rustup/issues/new"
```

The above is just a suggestion; it'd be still nice to use the `\` to keep the readability.